### PR TITLE
Feat/list custom fields cost centers

### PR DESCRIPTION
**Pull request by Josmar-jr**
#### What problem is this solving?
Now, it`s possible list custom fields by cost centers and not by organization

#### How to test it?

1. Navigate to [Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations/#/custom-fields)
2. Select the custom center at dropdown
3. Add new field on "ADD FIELD"
4. Edit the new field and name fill in and enable "Use no registration"	
5. Save the changes on "SAVE SETTINGS"
6. Go to storefront and click in "Organization SignUp"
7. Fill in all fields including the custom cost center field
8. Back to admin and access [Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations/#/requests)

[Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/organizations/#/requests)

#### Screenshots or example usage:

Screenshot 2024-07-11 at 11 44 01

### CODE REVIEW

**Comment by ArthurTriis1** [comment]
I'm trying to test the PR on the [workspace](https://josmarjr--b2bstore005.myvtex.com), but when I access the [Request Page](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-organizations/requests/e38d3ae7-42be-11ef-8452-0e5e9f59cfb3/), the Cost Center custom fields are not displayed, and the `getCostCentersByOrganizationId` function is not executed. Is this the correct workspace to test?

Screenshot 2024-07-15 at 12 43 23

**Comment by Josmar-jr** [comment]
Can we call to check this? @ArthurTriis1

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,44 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+t center
 
 ## [1.31.3] - 2024-04-22
 
 ### Fixed
 
-- Save add, edit and delete address within the Cost Center in admin
-
-### Fixed
-
```
Comment (ArthurTriis1) [inline CHANGELOG.md:47]: Some changes are being removed from the changelog, which is causing the checks to break. Could you please add those changes back and update the changelog?

**Comment by Josmar-jr** [comment]
I took the CHANGELOG from the master branch and added the feat from this PR, right?
@ArthurTriis1

**Comment by ArthurTriis1** [comment]
> I took the CHANGELOG from the master branch and added the feat from this PR, right? @ArthurTriis1

Yes, it is correct, but I still cannot see the custom fields for cost centers in the preview.

**Comment by ArthurTriis1** [comment]
> > I took the CHANGELOG from the master branch and added the feat from this PR, right? @ArthurTriis1
> 
> Yes, it is correct, but I still cannot see the custom fields for cost centers in the preview.

Checked It again, and now the Cost Center Fields are displayed, thank you!
Screenshot 2024-07-18 at 10 34 24

**Pull request merged**